src/object/conveyor_belt.cpp

```diff
@@ -128,6 +128,8 @@ ConveyorBelt::after_editor_set()
 
   if (m_length <= 0)
     m_length = 1;
+  m_col.m_bbox.set_size(32.0f * static_cast<float>(m_length), 32.0f);
```

Comment:
This is hardcoded and not dependent to the sprite hitbox. Conveyor belt overrides `update_hitbox()`, so that function should be called from here.